site.someones.Lonewolf.appdata.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>site.someones.Lonewolf</id>
+
+  <name>Lonewolf</name>
+  <summary>Organize and track your tasks with ease and flexibility.</summary>
+  <developer_name>Mario Aichinger</developer_name>
+
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+
+  <url type="homepage">https://github.com/aichingm/Lonewolf</url>
+  <url type="vcs-browser">https://github.com/aichingm/Lonewolf</url>
+  <url type="bugtracker">https://github.com/aichingm/Lonewolf/issues</url>
+
+  <supports>
+    <internet>offline-only</internet>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <display_length compare="ge">small</display_length>
+    <display_length compare="le">xlarge</display_length>
+  </supports>
+
+  <description>
+    <p>
+      Lonewolf is a productivity application based on the principles of Kanban.
+    </p>
+    <ul>
+      <li>Manages lists</li>
+      <li>Create cards</li>
+      <li>Organize cards with lables</li>
+      <li>Set due dates for yourself</li>
+      <li>Add attchments</li>
+      <li>Write descripotions of your tasks in markdown</li>
+      <li>Split up larger tasks with check lists</li>
+      <li>Keep your thoughts by adding comments for your future self</li>
+    </ul>
+  </description>
+
+  <launchable type="desktop-id">site.someones.Lonewolf.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/flathub/site.someones.Lonewolf/master/assets/screenshot_light_main.png</image>
+      <caption>Main window (light mode)</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/flathub/site.someones.Lonewolf/master/assets/screenshot_dark_main.png</image>
+      <caption>Main window (dark mode)</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/flathub/site.someones.Lonewolf/master/assets/screenshot_light_card.png</image>
+      <caption>Card Dialog</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/flathub/site.someones.Lonewolf/master/assets/screenshot_light_about.png</image>
+      <caption>About window</caption>
+    </screenshot>
+  </screenshots>
+
+  <releases>
+    <release type="stable" version="1.0.0" date="2023-09-25">
+      <description>
+        <p>Lonewolf's first release</p>
+      </description>
+    </release>
+  </releases>
+  <content_rating type="oars-1.0"/>
```

Comment:
```suggestion
  <content_rating type="oars-1.1"/>
```

Reply:
Interesting, is there a reason why not 1.0? My understanding was that 1.1 is only used if  works with GOME software but not other clients.

Reply:
1.1 is the latest. not sure where you get that idea

Reply:
The second dropdown on the generator page: https://hughsie.github.io/oars/generate.html

Reply:
it needs to be updated.